domains/drix10.is-cool.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Portfolio website",
+
+  "domain": "is-cool.dev",
+  "subdomain": "drix10",
+
+  "owner": {
+    "email": ""
```

Comment:
I removed the email you provided as it is invalid.

Reply:
Please replace with your email.

Reply:
updated email